tag-release.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+set -e
+
+usage() {
+    echo "Usage: $0 [-h] <SHA>"
```

Comment:
I can think of two ways to handle this:
- If we have a list of know tag prefixes (like `^validator`) I can just check for those but we run the risk of missing a named tag if someone ever wants to introduce a new one and forgets to update the script 
- Conversely I can pass a flag (like `-n <named>`) to indicate the primary argument is a named tag but then we risk applying a named tag and forgetting to use the flag

you have a preference @evanmcc ?

Reply:
went with the `-n <named>` option and _didn't_ gate the named tag on a list of "approved" tag prefixes since this script is not run automatically and the flag has to be supplied deliberately with no default value for that flag so `foobar1.6.0` would work just as well as `validator1.6.0` but if we want additional controls on only certain named tags triggering dependency tags then I would add them to the `if` on line 46:
```
if [[ ! -z $NAMED_TAG && $NAMED_TAG =~ ^validator* || $NAMED_TAG =~ ^seed* ]]; then
```

Reply:
Can you add -n to the usage text?